htdocs/compta/facture/prelevement.php

```diff
@@ -695,7 +695,7 @@
 
 	// Hook to change amount for other reasons, e.g. apply cash discount for payment before agreed date
 	$parameters = array('remaintopay' => $resteapayer);
-	$reshook = $hookmanager->executeHooks('finalizeAmountOfSupplierInvoice', $parameters, $object, $action); // Note that $action and $object may have been modified by some hooks
+	$reshook = $hookmanager->executeHooks('finalizeAmountOfCustomerInvoice', $parameters, $object, $action); // Note that $action and $object may have been modified by some hooks
```

Comment:
Depending if $type is 'bank-transfert' or not, the page is used commonly for vendor or supplier invoice. 
So i suggest name finalizeAmountOfInvoice